_redirects

```diff
@@ -17,6 +17,7 @@ layout: null
     {%- endfor %}
   {%- endif %}
   {%- if page.layout == 'product'  %}
+{{page.permalink}}/_edit     https://github.com/endoflife-date/endoflife.date/edit/master/{{page.name}}
```

Comment:
https://deploy-preview-3084--endoflife-date.netlify.app/almalinux/_edit says File not found. I think it must be the `path` instead of `name`.
```suggestion
{{page.permalink}}/_edit     https://github.com/endoflife-date/endoflife.date/edit/master/{{page.path}}
```

Reply:
https://deploy-preview-3084--endoflife-date.netlify.app/almalinux/_edit redirects for me:

```
curl -I https://deploy-preview-3084--endoflife-date.netlify.app/almalinux/_edit
HTTP/2 301
content-type: text/plain
date: Mon, 12 Jun 2023 08:43:43 GMT
location: https://github.com/endoflife-date/endoflife.date/edit/master/almalinux.md
server: Netlify
```